package.json

```diff
@@ -9,9 +9,7 @@
     "@adobe/aio-cli-lib-app-config": "^0.2.1",
     "@adobe/aio-lib-runtime": "^3.0.0",
     "@adobe/asset-compute-devtool": "^3.0.0",
-    "@oclif/command": "^1.8.0",
-    "@oclif/config": "^1.17.0",
-    "@oclif/plugin-help": "^3.2.2",
+    "@oclif/core": "^1.9.5",
```

Comment:
i assume oclif/core encompasses the rest of those plugins (config, command, plugin-help)?

Reply:
yeah, and @oclif/dev-cli